lib/admin/WP_Auth0_Admin_Advanced.php

```diff
@@ -595,6 +596,8 @@ public function render_social_twitter_secret( $args = array() ) {
 	public function render_social_facebook_key( $args = array() ) {
 		$this->render_social_key_field( $args['label_for'], $args['opt_name'] );
 		$this->render_field_description(
+			__( 'Facebook app key for the Social Amplification Widget. ', 'wp-auth0' ) .
+			__( 'The app used here needs to have "publish_actions" permission. ', 'wp-auth0' ) .
```

Comment:
Suggestion, if you think the links will last, link to facebook permissions' docs ?